privval/file_test.go

```diff
@@ -423,9 +426,11 @@ func TestVoteExtensionsAreSignedIfSignExtensionIsTrue(t *testing.T) {
 	// that validates against the vote extension sign bytes with the new
 	// extension, and does not validate against the vote extension sign bytes
 	// with the old extension.
-	vesb2 := types.VoteExtensionSignBytes("mychainid", vpb2)
+	vesb2, venrpsb2 := types.VoteExtensionSignBytes("mychainid", vpb2)
```

Comment:
Do we have a test here where the `NonRpExtension` is empty?

Reply:
extended test to cover this